src/utils/navigation.js

```diff
@@ -6,6 +6,10 @@ const getPagesSelector = () => {
   return [...document.querySelectorAll('#main-tabs > div > button')]
 }
 
+const getCurrentPagesSelector = () => {
+  return [...document.querySelectorAll('#main-tabs > div > button:not(.text-gray-500)')]
```

Comment:
I guess the correct selector should be simply:
```
document.querySelector('#main-tabs > div > button[aria-selected="true"]')
```
and then we don't need the array.

Reply:
Seems much better!